Automattic/jetpack
# My Jetpack: Add tiered pricing copy and update titles for Jetpack Search

#### Changes proposed in this Pull Request:

<img width="1094" alt="Screen Shot 2022-05-16 at 4 00 25 PM" src="https://user-images.githubusercontent.com/4044428/168695887-bd2f3f20-6cec-4057-9b03-6d2f0c3dfb5a.png">

Adds a snippet for tiered pricing when purchasing a search product via My Jetpack.

#### Jetpack product discussion



p9dueE-55u-p2#comment-7929 (cc @retrofox)

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply this change to your Jetpack-connected site (e.g. with the Boost plugin installed) **without a search subscription**.
* Navigate to My Jetpack (`wp-admin/admin.php?page=my-jetpack`) and click on "Purchase" within the Search card.
* Ensure that you see the new copy underneath the call-to-action button.
